rfhub2/db/repository/keyword_repository.py

```diff
@@ -24,6 +25,11 @@ def add(self, keyword: Keyword) -> Keyword:
         self.session.refresh(keyword)
         return keyword
 
+    def delete(self, keyword_id: int) -> int:
+        row_count = self._keywords.filter(Keyword.id == keyword_id).delete()
+        self.session.commit()
+        return row_count
```

Comment:
Why return row_count? Shouldn't it be single keywords with such id? From this method I assume that each keyword has unique id in general, not only in single collection?

Reply:
This way we can say if any row was deleted. If you will see the delete endpoint that uses this method, this way we can inform user that keyword with this id was properly removed if the row_count is 1 and respond with 404 if it was 0 to show that there was no record with this id to be removed. As you wrote, the keys are unique so this value should be 0/1